examples/poisson.pct.py

```diff
@@ -0,0 +1,236 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     custom_cell_magics: kql
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.2
+#   kernelspec:
+#     display_name: base
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Count data regression with the Poisson likelihood
+#
+# In this notebook we demonstrate how to perform inference for Gaussian process models
+# with non-Gaussian likelihoods via Markov chain Monte
+# Carlo (MCMC). We focus on a count data regression task here and use
+# [BlackJax](https://github.com/blackjax-devs/blackjax/) for sampling.
+
+# %%
+import blackjax
+import jax
+import jax.numpy as jnp
+import jax.random as jr
+import jax.scipy as jsp
+import jax.tree_util as jtu
+import matplotlib.pyplot as plt
+import optax as ox
+import tensorflow_probability.substrates.jax as tfp
+from jax.config import config
+from jaxtyping import Array, Float
+
+import gpjax as gpx
+
+# Enable Float64 for more stable matrix inversions.
+config.update("jax_enable_x64", True)
+tfd = tfp.distributions
+I = jnp.eye
```

Comment:
let's name this `identity_matrix` or simply use `jnp.eye`. Single letter variables are discouraged in PEP rules.